improver/cli/spot_extract.py

```diff
@@ -68,6 +69,10 @@ def process(
             Use to apply a lapse-rate correction to screen temperature data so
             that the data are a better match the altitude of the spot site for
             which they have been extracted.
+        fixed_lapse_rate (float):
+            If provided, use this fixed value as a lapse-rate for adjusting
+            the forecast values if apply_lapse_rate_correction is True. This
+            can be used instead of providing a lapse rate cube.
```

Comment:
Are there assumed units for this lapse rate? I assume that potentially it would be expected to match the units present in the temperature cube and the vertical displacement from the neighbour cube. It might be worth clarifying this. Also, it would be worth clarifying whether this value should include a minus e.g. -9.8 K/km or 9.8 K/km for a dry adiabatic lapse rate.

Reply:
Doc-string extended here and in apply-lapse-rate plugin.